subsys/bluetooth/host/audio/Kconfig

```diff
@@ -0,0 +1,109 @@
+# Bluetooth Audio configuration options
+
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_AUDIO
+	bool "Bluetooth Audio support"
+	help
+	  This option enables Bluetooth Audio support. The specific
+	  features that are available may depend on other features
+	  that have been enabled in the stack, such as Periodic
+	  Advertisement for Broadcast and L2CAP Dynamic Channel
+	  for Unicast.
+
+if BT_AUDIO
+
+config BT_ISO
+	# Virtual/hidden option
+	bool
+
+if BT_CONN
+
+config BT_AUDIO_UNICAST
+	bool "Bluetooth Unicast Audio Support"
+	select BT_SMP
+	select BT_L2CAP_DYNAMIC_CHANNEL
+	select BT_ISO
+	select BT_GATT_DYNAMIC_DB
+	select BT_GATT_CACHING
+	select BT_EATT
+	help
+	  This option enables support for Bluetooth Unicast Audio using
+	  Isochronous channels.
+
+if BT_AUDIO_UNICAST
+
+config BT_MAX_ISO_CONN
+	int "Maximum number of simultaneous ISO connections"
```

Comment:
Is the in any way depending on `CONFIG_BT_MAX_CONN`?

Reply:
I guess the minimal would have to be based on CONFIG_BT_MAX_CONN, but we can end up having more than one ISO per ACL pair.

Reply:
I'd suggest to set the minimum to CONFIG_BT_MAX_CONN, so that we can catch mis-configurations at compile times rather than runtime. If there's a maxmium number of ISO per ACL, then I'd suggest also using CONFIG_BT_MAX_CONN times the max number of ISO per ACL as the maximum here

Reply:
Done.